demo/python/remote-control/client/client.py

```diff
@@ -24,14 +25,14 @@ def run(channel_addr, sym_key_file, priv_key_file, cert_file):
     enclave_reference.add_key()
 
     print("Creating training matrix")
-    dtrain = xgb.DMatrix({username: HOME_DIR + "demo/data/agaricus.txt.train.enc"}, encrypted=True)
+    dtrain = xgb.DMatrix({username: HOME_DIR + "demo/python/remote-control/data/train.enc"}, encrypted=True)
```

Comment:
Drop `encrypted=True` here and elsewhere. This is the default, plus we do not support `encrypted=False` at the moment.

Reply:
Fixed